packages/massa-proto-cli/src/protobuf.ts

```diff
@@ -68,12 +68,26 @@ export async function getProtoFunction(protoPath: string): Promise<ProtoFile> {
     .filter(([, value]) => value)
     .map(([name, field]) => ({
       name,
-      type: (field as { type: string; id: number }).type,
+      type:
```

Comment:
I think this is equivalent and more readable

```ts
    .map(([name, field]) => {
      const fielType = (field as { type: string; id: number }).type;
      const fieldRule =
        (field as { rule: string; type: string; id: number }).rule ===
        'repeated'
          ? '[]'
          : '';

      return {
        name,
        type: fielType + fieldRule,
      };
    });
```

Reply:
code updated !